schemes/chemistry/prescribed_ozone.F90

```diff
@@ -0,0 +1,170 @@
+! read and provide prescribed ozone for radiation
+! this is a simple example of a CCPP scheme using the CAM-SIMA tracer_data utility.
+!
+! Based on original CAM version from: Francis Vitt
+module prescribed_ozone
+  use ccpp_kinds,  only: kind_phys
+
+  ! CAM-SIMA host model dependency to read chemistry data.
+  use tracer_data, only: trfile     ! data information and file read state.
+  use tracer_data, only: trfld      ! tracer data container.
+
+  implicit none
+  private
+
+  ! public CCPP-compliant subroutines
+  public :: prescribed_ozone_init
+  public :: prescribed_ozone_run
+
+  ! fields to store tracer_data state and information.
+  type(trfld), pointer :: tracer_data_fields(:)
+  type(trfile)         :: tracer_data_file
+
+  ! parameters for ozone
+  ! TODO - this should use a centralized chemistry species database
+  real(kind_phys),  parameter :: ozone_mw = 47.9981995_kind_phys
+
+  ! namelist options
+  logical                     :: has_prescribed_ozone = .false.
+  character(len=8), parameter :: ozone_name = 'ozone' ! name of the output field
+
+contains
+
+!> \section arg_table_prescribed_ozone_init  Argument Table
+!! \htmlinclude prescribed_ozone_init.html
+  subroutine prescribed_ozone_init( &
+    amIRoot, iulog, &
+    fld_name, filename, filelist, datapath, &
+    data_type, &
+    cycle_yr, fixed_ymd, fixed_tod, &
+    errmsg, errflg)
+
+    use cam_history, only: history_add_field
+    use tracer_data, only: trcdata_init
+
+    logical,            intent(in)  :: amIRoot
+    integer,            intent(in)  :: iulog
+
+    ! input fields from namelist to initialize tracer_data
+    character(len=*),   intent(in)  :: fld_name   ! field name
+    character(len=*),   intent(in)  :: filename   ! input filename
+    character(len=*),   intent(in)  :: filelist   ! input filelist
+    character(len=*),   intent(in)  :: datapath   ! input datapath
+    character(len=*),   intent(in)  :: data_type  ! data type
+    integer,            intent(in)  :: cycle_yr   ! cycle year
+    integer,            intent(in)  :: fixed_ymd  ! fixed year-month-day (YYYYMMDD) [1]
+    integer,            intent(in)  :: fixed_tod  ! fixed time of day [s]
+    character(len=512), intent(out) :: errmsg
```

Comment:
As discussed in the RRTMGP PR, let's convert all of the `errmsg` outputs to `len=*` instead of a specified number:


```suggestion
    character(len=*), intent(out) :: errmsg
```

Reply:
Thanks, updated throughout